ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/PlatformCvesOverviewPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {
+    PageSection,
+    Title,
+    Divider,
+    Flex,
+    FlexItem,
+    Card,
+    CardBody,
+} from '@patternfly/react-core';
+
+import PageTitle from 'Components/PageTitle';
+import useURLStringUnion from 'hooks/useURLStringUnion';
+import useURLPagination from 'hooks/useURLPagination';
+import useURLSearch from 'hooks/useURLSearch';
+import { getHasSearchApplied } from 'utils/searchUtils';
+
+import EntityTypeToggleGroup from '../../components/EntityTypeToggleGroup';
+import { parseWorkloadQuerySearchFilter } from '../../utils/searchUtils';
+import { platformEntityTabValues } from '../../types';
+
+import CVEsTableContainer from './CVEsTableContainer';
+import ClustersTableContainer from './ClustersTableContainer';
+
+function PlatformCvesOverviewPage() {
+    const [activeEntityTabKey] = useURLStringUnion('entityTab', platformEntityTabValues);
+    const { searchFilter } = useURLSearch();
+    const pagination = useURLPagination(20);
+
+    // TODO - Need an equivalent function implementation for filter sanitization for Platform CVEs
+    const querySearchFilter = parseWorkloadQuerySearchFilter(searchFilter);
+    const isFiltered = getHasSearchApplied(querySearchFilter);
+
+    function onEntityTabChange() {
+        pagination.setPage(1);
+        // TODO - set default sort here
+    }
+
+    // TODO - needs to be connected to a query
```

Comment:
Commenting as a thought for other work in the future, knowing that Workload CVEs already has this pattern:
* Somewhat similar to classic side panel idiom which requires plural request even for deep link to singular entity, bookmark to any tab/toggle requires request for all counts.
* Although there is `pluralize` for the one case, it would be a globalization problem, and also is small edge case, because technical writer will need to refer to the plural entity without count: Click the **Clusters** tab/toggle. As an example how to prevent both problems, Gmail has **Inbox** followed by count.
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/components/EntityTypeToggleGroup.tsx#L33
* PatternFly guidelines for Tabs nor ToggleGroup have count entity idiom. Precedent in other OpenShift UIs?

Reply:
These are all good points, and I think worthwhile to bring up with the UX team. There were conversations a while back about moving away from the "count" pattern altogether, mostly for performance/scale reasons on the BE, but I wasn't really a part of them. This is especially relevant to the first point, as well need the full count of 3 entities here that the user may not even view.